Makefile

```diff
@@ -5,29 +5,43 @@ all: quality test
 clean:
 	-rm -rf node_modules/
 
-test: test-py
+test: test-py test-js
 
 test-py:
 	nosetests video_xblock --with-coverage --cover-package=video_xblock
 
+test-js:
+	export DISPLAY=:99.0
+	sh -e /etc/init.d/xvfb start
+	karma start video_xblock/static/video_xblock_karma.conf.js
+	sh -e /etc/init.d/xvfb stop
+
 quality: quality-py quality-js
 
 quality-py:
 	pep8 . --format=pylint --max-line-length=120
 	pylint -f colorized video_xblock
+	pydocstyle -e
 
 quality-js:
 	eslint video_xblock/static/js/
 
 deps:
-	pip install -r requirements.txt
-	bower install
+	pip install --process-dependency-links -r requirements.txt
```

Comment:
@polesye do you mean `python setup.py install`?
I'm not sure how well this going to play with Travis.